cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -281,7 +281,7 @@ function is_leader() {
     return 1
     ;;
 
-  "${HOSTNAME}"_*)
+  "${HOSTNAME}"_* | "${HOSTNAME}"---*)
```

Comment:
Should the postfix hash match be for only the chars allowed in the hash instead of `*`?

Reply:
Not opposed to it, though I'm trying to make the minimal amount of changes to prevent in case people make assumptions about this "bug/feature".

Reply:
I'm not sure a wider regex prevents assumptions here.

Reply:
Updated, still need to do some testing but this regex `[a-zA-Z0-9-]` captures uuid chars. Had to switch to using an if statement for more complex regex matches.